packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -70,69 +71,73 @@
 .secondaryVariant {
   border-color: var(--ColorBorderSecondary);
   background-color: var(--ColorBackgroundSecondary);
-  color: $color-primary-base;
+  color: var(--ColorTextSecondary);
 
   &:hover {
     border-color: var(--ColorBorderSecondaryHover);
     background-color: var(--ColorBackgroundSecondaryHover);
-    color: $white;
+    color: var(--ColorTextSecondaryHover);
   }
 
   &:focus {
     border-color: var(--ColorBorderSecondaryFocus);
     background-color: var(--ColorBackgroundSecondaryFocus);
-    color: $color-primary-base;
+    color: var(--ColorTextSecondaryFocus);
   }
 
   &:disabled {
     border-color: var(--ColorBorderSecondaryDisabled);
     background-color: var(--ColorBackgroundSecondaryDisabled);
-    color: $color-primary-light;
+    color: var(--ColorTextSecondaryDisabled);
   }
 }
 
 .dangerVariant {
-  background-color: $color-danger-base;
+  background-color: var(--ColorBackgroundDanger);
 
   &:hover {
-    border-color: $color-danger-dark;
-    background-color: $color-danger-dark;
+    border-color: var(--ColorBorderDangerHover);
+    background-color: var(--ColorBackgroundDangerHover);
   }
 
   &:focus {
-    @include outline($focus-ring-danger);
+    @include outline(var(--ColorFocusDanger), var(--FocusOutlineWidth));
 
-    background-color: $color-danger-dark;
+    background-color: var(--ColorBackgroundDangerFocus);
   }
 
   &:disabled {
-    border-color: $color-danger-light;
-    background-color: $color-danger-light;
+    border-color: var(--ColorBorderDangerDisabled);
+    background-color: var(--ColorBackgroundDangerDisabled);
   }
 }
 
 .dismissVariant,
 .dismissInvertedVariant {
-  padding-block: ($spacing-xs-em * 0.5);
-  padding-inline: ($spacing-xs-em * 0.5);
+  padding-block: calc(var(--SpacePaddingBlockDismiss) * 0.5);
+  padding-inline: calc(var(--SpacePaddingInlineDismiss) * 0.5);
   border: 0;
-  background-color: transparent;
```

Comment:
The ability to theme these properties will open Dismiss up to additional use cases. I'm not worried about the implementation, but it'd be good to have some guidelines for what makes a property static or private. This PR has some good starting points:

With `white-space` above, a static value ensures the proper behavior. ✅ 
With `BorderColorPrimary` being added above, we offer unplanned variance on our designs, but nothing is at risk. ✅ 

These properties sit somewhere in between. Theming them would allow this button to be used in ways/contexts we didn't design it for. That's not an issue for customers or external users, but it might present a risk for Oktanauts.

Reply:
@edburyenegren-okta yes you are thinking about this in all the rights ways. Theme variables for components have a large amount of hidden complexity. API design discussions, design intention discussions, and lots of tight coupling to the associated javascript code and style modules that needs to be carefully managed to keep in sync and avoid leaking into the public interface.

For now, we need to open up more flexibility for SIW, but we should also favor a tiny surface area for our API when possible. That means never exposing theme variables that aren't necessary and allowing teams to come to us to ask for them. Long term this is where I want us to be.

SIW is a tricky exception to this rule because customers already are accustom to being able to update anything they want and we don't want to encourage them to continue using CSS cascade based style overrides, instead making use of our public API here.